yamls/baselines/colbert/bert-base-uncased.yaml

```diff
@@ -0,0 +1,31 @@
+# This YAML is built to work with the `sequence_classification.py` starter script!
+#
+#   Follow the instructions in that script to modify the `build_my_dataloader` function
+#   and fine-tune a BERT model on your own dataset!
+#
+#
+#   Note that some of the fields in this template haven't been filled in yet.
+#   Please resolve any empty fields before launching!
+
+# Run Name
+run_name: finetune-hf-bert
+
+seed: 42
+model_name_or_path: bert-base-uncased
+train_dataset_id: bclavie/msmarco-2m-triplets-colbert-format
+n_gpu: 1
+tmp_dir: ./tmp_colbert
+debug: false
+
+train_params:
+  lr: 3e-5
+  use_ib_negatives: true
+  bsize: 32
+
+eval_datasets:
```

Comment:
I think this set of evals is nice, all the shorter BEIR ones.  It would be nice if there was an MTEB lite, but I don't think it's out yet.  This is a great start.  

I think ideally we'd get one that is more standard "web query" eval but NQ and MSMarco are too large for this to be quick rn.

Reply:
> I think ideally we'd get one that is more standard "web query" eval but NQ and MSMarco are too large for this to be quick rn.

I was thinking we could do the MS Marco passage reranking task, but I'm not sure the results on that are super informative (it seems pretty heavily saturated score-wise)

Reply:
Yes, these are the datasets that are not prohibitively expensive to test on.
If we want to run on other domains, we could use/create subsamples (e.g, only using documents that have a qrel or sample a part of the documents). Again, it depends if this is for quick evaluation to compare different models we trained (e.g, ablations or so) or if it's for reporting.

Reply:
Quick evaluations for now. We can refine later for reporting if needed!